nixos/lib/systemd-lib.nix

```diff
@@ -524,7 +524,9 @@ rec {
           # Stupid misc. symlinks.
           ln -s ${cfg.defaultUnit} $out/default.target
           ln -s ${cfg.ctrlAltDelUnit} $out/ctrl-alt-del.target
-          ln -s rescue.target $out/kbrequest.target
+          ${optionalString (cfg.kbrequestUnit != null) ''
+            ln -s ${cfg.kbrequestUnit} $out/kbrequest.target
+          ''}
```

Comment:
possibly remove kbrequest.target in its entirety? see https://github.com/NixOS/nixpkgs/pull/419015#pullrequestreview-3111792107

```suggestion
```